### PR TITLE
Improve loading on app startup

### PR DESCRIPTION
**Pull request by henrygoodman**
Improve loading models on app startup, tweak minimum tokens to 1 for shorter inputs

### CODE REVIEW

**Review by Rgonzales4 — APPROVED** [review]
LGTM, after this is merged I'll merge these changes in and add to main.py for the sentiment PR👌

**Review by stephenb-coding — APPROVED** [review]

**Pull request merged**
